Komodo integration test fails for LSF driver

**Describe the bug**
This should not fail:

```
=================================== FAILURES ===================================
____________________________ test_submit[LsfDriver] ____________________________

driver = <ert.scheduler.lsf_driver.LsfDriver object at 0x7f1d9d8ee0d0>
tmp_path = PosixPath('/private/f_scout_ci/pytest-tmp/tmp.2xNXKq9SKY/test_submit_LsfDriver_0')

    @pytest.mark.integration_test
    async def test_submit(driver, tmp_path):
        await driver.submit(0, "sh", "-c", f"echo test > {tmp_path}/test")
        await poll(driver, {0})
    
>       assert (tmp_path / "test").read_text(encoding="utf-8") == "test\n"

integration_tests/scheduler/test_generic_driver.py:53: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/usr/lib64/python3.8/pathlib.py:1236: in read_text
    with self.open(mode='r', encoding=encoding, errors=errors) as f:
/usr/lib64/python3.8/pathlib.py:1222: in open
    return io.open(self, mode, buffering, encoding, errors, newline,
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = PosixPath('/private/f_scout_ci/pytest-tmp/tmp.2xNXKq9SKY/test_submit_LsfDriver_0/test')
name = '/private/f_scout_ci/pytest-tmp/tmp.2xNXKq9SKY/test_submit_LsfDriver_0/test'
flags = 524288, mode = 438

    def _opener(self, name, flags, mode=0o666):
        # A stub for the opener argument to built-in open()
>       return self._accessor.open(self, flags, mode)
E       FileNotFoundError: [Errno 2] No such file or directory: '/private/f_scout_ci/pytest-tmp/tmp.2xNXKq9SKY/test_submit_LsfDriver_0/test'

/usr/lib64/python3.8/pathlib.py:1078: FileNotFoundError
=========================== short test summary info ============================
FAILED integration_tests/scheduler/test_generic_driver.py::test_submit[LsfDriver] - FileNotFoundError: [Errno 2] No such file or directory: '/private/f_scout_ci/pytest-tmp/tmp.2xNXKq9SKY/test_submit_LsfDriver_0/test'
```

https://github.com/equinor/komodo-releases/actions/runs/8273452506/job/22638541889

--- a/tests/integration_tests/scheduler/test_generic_driver.py
+++ b/tests/integration_tests/scheduler/test_generic_driver.py
@@ -47,6 +47,7 @@ def driver(request, pytestconfig, monkeypatch, tmp_path):
 
 @pytest.mark.integration_test
 async def test_submit(driver, tmp_path):
+    os.chdir(tmp_path)
     await driver.submit(0, "sh", "-c", f"echo test > {tmp_path}/test")
     await poll(driver, {0})
 
@@ -54,6 +55,7 @@ async def test_submit(driver, tmp_path):
 
 
 async def test_submit_something_that_fails(driver, tmp_path):
+    os.chdir(tmp_path)
     finished_called = False
 
     expected_returncode = 42
@@ -76,7 +78,8 @@ async def test_submit_something_that_fails(driver, tmp_path):
     assert finished_called
 
 
-async def test_kill(driver):
+async def test_kill(driver, tmp_path):
+    os.chdir(tmp_path)
     aborted_called = False
 
     expected_returncodes = [1]

--- a/tests/integration_tests/scheduler/test_lsf_driver.py
+++ b/tests/integration_tests/scheduler/test_lsf_driver.py
@@ -21,6 +21,7 @@ def mock_lsf(pytestconfig, monkeypatch, tmp_path):
 
 @pytest.mark.parametrize("explicit_runpath", [(True), (False)])
 async def test_lsf_info_file_in_runpath(explicit_runpath, tmp_path):
+    os.chdir(tmp_path)
     driver = LsfDriver()
     (tmp_path / "some_runpath").mkdir()
     os.chdir(tmp_path)
@@ -41,7 +42,8 @@ async def test_lsf_info_file_in_runpath(explicit_runpath, tmp_path):
     ).keys() == {"job_id"}
 
 
-async def test_job_name():
+async def test_job_name(tmp_path):
+    os.chdir(tmp_path)
     driver = LsfDriver()
     iens: int = 0
     await driver.submit(iens, "sh", "-c", "sleep 99", name="my_job")
@@ -80,12 +82,15 @@ async def test_submit_to_named_queue(tmp_path, caplog):
         ([256, 0]),  # return codes are 8 bit.
     ],
 )
-async def test_lsf_driver_masks_returncode(actual_returncode, returncode_that_ert_sees):
+async def test_lsf_driver_masks_returncode(
+    actual_returncode, returncode_that_ert_sees, tmp_path
+):
     """actual_returncode is the returncode from job_dispatch.py (or whatever is submitted)
 
     The LSF driver is not picking up this returncode, it will only look at the
     status the job obtains through bjobs, which is success/failure.
     """
+    os.chdir(tmp_path)
     driver = LsfDriver()
 
     async def finished(iens, returncode, aborted):

